### DISCUSSION TITLE
`as2` branch progress

### DISCUSSION THREAD

**Discussion by CRKatri** [ProcursusTeam/Procursus#714] [General]
What is left to do for Apple Silicon? It seems that it should be finishing soon, but I'm not really in the loop of the going-ons.

**Comment by asdfugil** [comment]
there's stuff on macOS that depends on firmware 14.0
I have to create a dummy firmware package to workaround it.
but that's not a proper fix

**Reply by CRKatri** [reply]
Just use .control.macosx for that
